primitives.js

```diff
@@ -0,0 +1,2 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+module.exports = require('./lib/primitives')
```

Comment:
We can probably delete this file and `/native.js`, since we have a `native/` and `primitives/` folder now

cc @patrick91 @mxstbr